Pow.java

```diff
@@ -0,0 +1,32 @@
+/*
+
+
+Please explain why it is O( Log n)
+
+I am not good with time  complexities
+
```

Comment:
Regarding Time Complexity : for example if we want to calculate 2^8 then instead of multiplying 2 (8 times) which is O(N) complexity we can reduce computation by half each and everytime by (2^4)^2 -> ((2^2)^2))^2 
i.e 2*2 = 4  -> 4*4 = 16 -> 16*16 = 256 which we got the result just by 3 computations i.e half of the input power
so it is O(logN)